cypress/TestCases/FireboltCertification/UserGrants.feature

```diff
@@ -0,0 +1,778 @@
+Feature: UserGrants
+
+    # If lifespan value is set to once, grant is requested for every attempt.
+    # Case-1: Validating by allowing the grant
+    # Case-2: Validating by denying the grant (using pinchallenge and acknowledge).
+    @Usergrants @coreSDK @sdk @transport @requiresPlatformImplementation
+    Scenario: UserGrants.Grant - Positive Scenario: Validate Capabilities with denied (pinChallenge) and lifespan once
+        Given the environment has been set up for 'userGrants' tests
+        And 3rd party 'certification' app is launched
+        # Case-1
+        And Framework registers 'pinChallenge' test provider
+        And User 'starts' recording lifecycle history for '3rd party app'
+        When '3rd party app' invokes the 'Firebolt' API to 'get postalcode'
+        Then 'Firebolt' platform responds with '20850 for localization postalcode'
+        When '3rd party app' invokes the 'Firebolt' API to 'check if capabilities is granted for postalcode'
```

Comment:
how do you know that 20850 is returned for postalcode ?

Reply:
Updated the key name and verified other key names as well. It seems the existing key names are being used . For example, `washington for localization locality` . Do we need to change the key name there as well?

Reply:
yes. make the TC generic. Or else, add a setter for the same to ensure that the we are setting the proper values before making a get call

Reply:
Updated